docs/application/dotnet/guides/nui/view.md

```diff
@@ -1,75 +1,133 @@
+
+- [View.UpFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_UpFocusableView)
+
+- [View.DownFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_DownFocusableView)
+
+- [View.LeftFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_LeftFocusableView)
+
+- [View.RightFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_RightFocusableView)
+
+When an arrow key is pressed and the current `View` has next `FocusableView` for that direction, the system passes focus to the next `FocusableView`. However, if the current `View` does not have the next `FocusableView` for the direction or the next `ForcusableView` is not on the window, the property returns `NULL` and no change occurs.
 
-![TextLabel object with a red background](./media/background_textlabel.png)
 
 <a name="3"></a>
-## Setting the Background Image
+## View Event
 
-You can set a background image of a view:
+`View` provides a variety of `event`s that are commonly used by the components. The following table lists the basic events provided by `View`:
 
+**Table: View events**
+
+| Event               | Description                                               |
+| ------------------- | --------------------------------------------------------- |
+| `KeyEvent`          | Triggered when the key input is received.                   |
+| `TouchEvent`        | Triggered when the touch input is received.                 |
+| `HoverEvent`        | Triggered when the hover input is received.                 |
+| `WheelEvent`        | Triggered when the wheel input is received.                 |
+| `FocusGained`       | Triggered when the control gets the key input focus.        |
+| `FocusLost`         | Triggered when the control loses key input focus.           |
+| `Relayout`          | Triggered after the size has been set on View during relayout. |
+| `LayoutDirectionChanged` | Triggered when the layout direction property of View or its parent View is changed. |
+| `AddedToWindow`     | Triggered after View has been connected to the window.  |
+| `RemovedFromWindow` | Triggered after View has been disconnected from the window. |
+| `VisibilityChanged` | Triggered when the visible property of this or a parent View is changed. |
+| `ResourcesLoaded`   | Triggered after all the resources required by View are loaded and ready. |
+
+You can create custom `callback` methods with event handler for each `event`.
+
+Input `Event`s such as `KeyEvent` and `TouchEvent` are first received on the current focused `View` to be handled. However, if the focused `View` is not having a proper event handler, then the `Event` is delivered to its parent `View` iteratively until it can be consumed.
```

Comment:
In line 80, it is `event`. Here we are using `Event`.
Is there a specific reason for the change?
Please verify.

Consider the same for line 103.

Reply:
Thank you for your correction.

Reply:
@bshqa
I cannot see the applied change. 
Could you please verify the comment.